Cargo.toml

```diff
@@ -160,6 +160,9 @@ lto = true
 codegen-units = 1
 strip = true
 
+[profile.production]
+inherits = "release"
+
```

Comment:
The gh action `paritytech/try-runtime-gha@v0.1.0` looks for production profile, added production to inherit release